source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/list/__tests__/page.ts

```diff
@@ -0,0 +1,16 @@
+
+import {
+  interactor, hasClass,
+  clickable
+} from '@bigtest/interactor';
+
+@interactor
+export class UserInteractor {
+  constructor(selector?: string) { }
+
+  clickLockUser = clickable('[data-test-toggle-lock]:first-child');
+  unlockUser = hasClass('[data-test-toggle-lock]:first-child','checked');
```

Comment:
this property name is a command, but the value represents a status. So, the property name should be updated to represent the value :)